src/cobalt_model/pagination_config.rs

```diff
@@ -121,13 +121,15 @@ impl PaginationConfigBuilder {
             return None;
         }
         let per_page = per_page.unwrap_or(DEFAULT_PER_PAGE);
-        let permalink = permalink.unwrap_or_else(|| DEFAULT_PERMALINK.to_owned());
+        let permalink_suffix =
+            permalink_suffix.unwrap_or_else(|| DEFAULT_PERMALINK_SUFFIX.to_owned());
         let order = order.unwrap_or(SortOrder::Desc);
         let sort_by = sort_by.unwrap_or_else(|| vec![DEFAULT_SORT.to_owned()]);
         Some(PaginationConfig {
             include,
             per_page,
-            permalink,
+            front_permalink: permalink.to_string(),
```

Comment:
`to_string` I assume your intention is more so `to_owned`?

They both work, the latter just clarifies intent